main.go

```diff
@@ -1,7 +1,14 @@
 package main
 
-import "fmt"
-
 func main() {
-	fmt.Println("cel-scanner-poc")
+	// read rule as input
```

Comment:
We could hard code to the examples in https://github.com/rhmdnd/cel-scanner-poc/pull/2/files just to get something working initially.